apps/db/lib/friend_request/friend_request.ex

```diff
@@ -0,0 +1,40 @@
+defmodule DB.FriendRequest do
```

Comment:
Sorry for not getting to this sooner, I'm curious about the decision to call it a `FriendRequest`. Do we anticipate ever having more info on this table that might make `Friendship` a more descriptive name? I think it's pretty clear either way, but something to think about